src/components/common/Actions.vue

```diff
@@ -171,19 +171,19 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
     const paymentToken = filingComplete?.header?.paymentToken
     if (paymentToken) {
       const isPaymentActionRequired: boolean = filingComplete.header?.isPaymentActionRequired
-      const dashboardUrl = sessionStorage.getItem('DASHBOARD_URL')
+      const returnUrl = sessionStorage.getItem('DASHBOARD_URL') + this.getBusinessId +
+        `?filing_id=${this.getFilingId}`
```

Comment:
We can use getFilingId here because this is a correction, which started out from a draft (which has a filing ID).

Also, line 162 would fail if there was no filing ID.